test/dotnet-grpc.Tests/CommandBaseTests.cs

```diff
@@ -450,4 +470,57 @@ public void IsUrl_ChecksUrlValidity(string url, bool isUrl)
         private Project CreateIsolatedProject(string path)
             => Project.FromFile(path, new ProjectOptions { ProjectCollection = new ProjectCollection() });
     }
+
+    [TestFixture]
+    public class CommandBaseRemoteFileTests : TestBase
+    {
+        [OneTimeSetUp]
+        public void InitializeRemoteFile()
```

Comment:
Why have any of this as fields on the test class? Just have a helper method that someone can use to create an HttpClient with the expected content. That is what all the gRPC client tests do.

Reply:
I was trying to minimize code duplication.